src/store/state/state-model.ts

```diff
@@ -50,17 +50,55 @@ export const stateModel: StateModelIF = {
       isValidAssociationType: true,
       isValidCreateSpecialResolution: true
     },
-    flagsReviewCertify: {
```

Comment:
flagsReviewCertify is now deleted and has been replaced with smaller a state models that are specific to each filing